recipe/eqt_env.yml

```diff
@@ -1,9 +1,8 @@
-# This creates an environment with all of the prerequisites for eqt
 name: eqt_env
 channels:
   - conda-forge
 dependencies:
-  - python
+  - python=<3.12
```

Comment:
Based on your tests I believe this should be < (not equal) to 3.12. Do you agree? i.e. `python <3.12`

Reply:
I was working off an example formatted like `python={version}`, so `python=<3.12` would mean Python version is equal to {<3.12} - I can see how this might cause confusion so I've changed it to match the format used in the CIL-GUI recipes, i.e. `python {version}` >>> `python <3.12`